README.md

```diff
@@ -9,6 +9,7 @@ The dates so far have been:
 | Bootcamp 2024 | July 15-19, 2024 | Lewis Library, Princeton, NJ |
 | Bootcamp 2025 | July 14-18, 2025 | Jadwin Hall, Room 407, Princeton, NJ |
 
+Cats are great.
```

Comment:
I think it's really important to bring attention to this, thank you for adding it.

Can we include some reasons why cats are great just in case anyone is confused?